### DISCUSSION TITLE
Couldn't find Transmission's web interface files!

### DISCUSSION THREAD

**Discussion by Oleg-Chashko** [transmission/transmission#2519] [Q&A]
- macOS Big Sur: v.11.6 (20G165)
- Transmission: 3.00+850b3caa61

http://localhost:9091/transmission/web/

404: Not Found
Couldn't find Transmission's web interface files!
Users: to tell Transmission where to look, set the TRANSMISSION_WEB_HOME environment variable to the folder where the web interface's index.html is located.
Package Builders: to set a custom default at compile time, #define PACKAGE_DATA_DIR in libtransmission/platform.c or tweak tr_getClutchDir() by hand.

**Comment by ckerr** [comment]
Yep, this is likely because you're running a build that you made yourself, rather than one that was installed with an installer. That's OK.

The way I usually get around this is to set an environment variable `TRANSMISSION_WEB_HOME` set to /path/to/where/you/git/cloned.../transmission/web/public_html

If you prefer, you can also hardcode that path in tr_getClutchDir().

**Comment by Oleg-Chashko** [comment]
_Yep, this is likely because you're running a build that you made yourself._

No, the assembly was taken from here:
https://build.transmissionbt.com/job/trunk-mac/

**Comment by amandamana** [comment]
Any solution to this ongoing problem with those trunk builds?
../Contents/Resources/web/public_html/index.html exists inside the .app

**Comment by GaryElshaw** [comment]
I had the same issue and then worked out (or didn't?) that you have to install the Transmission CLI app as well, which i did with [homebrew](https://brew.sh/), then it was just a matter of brew install transmission-cli and going from there.

'brew services start transmission-cli' open the web interface from the Transmission preferences window. And then 'brew services stop transmission-cli' if you want it to stop running in the background.

**Comment by amandamana** [comment]
Line 401 in platform.cc reads:
`#ifdef BUILD_MAC_CLIENT /* on Mac, look in the Application Support folder first, then in the app bundle. */`
so I tried creating a symbolic link in ~/Library/Application Support/Transmission/ for both the public_html and the web folders pointing to originals in the app bundle. But the web interface still failed to find its files. I also tried adding symbolic links in /usr/local/share/transmission/ with no effect.
